static/app/api.tsx

```diff
@@ -432,11 +444,7 @@ export class Client {
     // GET requests may not have a body
     const body = method !== 'GET' ? data : undefined;
 
-    const headers = new Headers({
-      Accept: 'application/json; charset=utf-8',
-      'Content-Type': 'application/json',
-      ...options.headers,
-    });
+    const headers = new Headers(this.headers);
```

Comment:
Im not sure anything actually uses this option but this now differs from the original implementation where the JSON headers were always present unless overwritten whereas this uses them as a fallback if no headers were specified. Do we need to check if this will affect any API calls?

Reply:
No, if you see above in the constructor, we init headers to the same value as before actually